go/extractor/dbscheme/tables.go

```diff
@@ -1150,6 +1150,20 @@ var ComponentTypesTable = NewTable("component_types",
 	EntityColumn(TypeType, "tp"),
 ).KeySet("parent", "index")
 
+// ComponentTagsTable is the table associating composite types with their component types' tags
+var ComponentTagsTable = NewTable("component_tags",
+	EntityColumn(CompositeType, "parent"),
```

Comment:
```suggestion
	EntityColumn(StructType, "parent"),
```
Tags only exist on fields of structs, so I don't see why we should make this table more general than that. (Various names should change as well, of course.)